src/main/java/frc/team4909/robot/operator/generic/BionicJoystick.java

```diff
@@ -45,38 +46,38 @@ public double getSensitiveAxis(BionicAxis axis) {
 
     }
 
-    public void povActive(BionicPOV povAngle, Commandable commandable) {
+    public void povActive(BionicPOV povAngle, InstantCommand command) {
         BionicJoystickPOVButton newPov = new BionicJoystickPOVButton(this, povAngle.getNumber());
 
-        newPov.whenActive(commandable);
+        newPov.whenActive(command);
     }
 
     /**
      * @param button      Button to Create Handler For
      * @param commandable Returns a Commandable that can be used by the operator and autonomous CommandGroups
      */
-    public void buttonPressed(BionicButton button, Commandable commandable) {
+    public void buttonPressed(BionicButton button, InstantCommand command) {
         JoystickButton newButton = new JoystickButton(this, button.getNumber());
 
-        newButton.whenPressed(commandable);
+        newButton.whenPressed(command);
     }
 
     /**
      * @param axis        Axis to Create Handler For
      * @param threshold   Minimum Threshold to Trigger Command
      * @param commandable Returns a Commandable that can be used by the operator and autonomous CommandGroups
      */
-    public void buttonPressed(BionicAxis axis, double threshold, Commandable commandable) {
+    public void buttonPressed(BionicAxis axis, double threshold, InstantCommand command) {
         BionicJoystickAxisButton newButton = new BionicJoystickAxisButton(this, axis.getNumber(), threshold);
 
-        newButton.whenActive(commandable);
+        newButton.whenActive(command);
     }
 
     /**
      * @param button      Button to Create Handler For
      * @param commandable Returns a Commandable that can be used by the operator and autonomous CommandGroups
      */
-    public void buttonHeld(BionicButton button, Commandable commandable) {
+    public void buttonHeld(BionicButton button, InstantCommand commandable) {
```

Comment:
This doesn't seem right. For `buttonPressed`, an `InstantCommand` seems appropriate, but for `buttonHeld`, shouldn't the `Command` run as long as the button is held?